rocketpy/motors/solid_motor.py

```diff
@@ -355,9 +370,70 @@ class Function. Thrust units are Newtons.
 
         self.evaluate_geometry()
 
-        # Initialize plots and prints object
         self.prints = _SolidMotorPrints(self)
         self.plots = _SolidMotorPlots(self)
+        self.propellant_I_11_from_propellant_CM = self.propellant_I_11
+        self.propellant_I_22_from_propellant_CM = self.propellant_I_22
+        self.propellant_I_33_from_propellant_CM = self.propellant_I_33
+        self.propellant_I_12_from_propellant_CM = self.propellant_I_12
+        self.propellant_I_13_from_propellant_CM = self.propellant_I_13
+        self.propellant_I_23_from_propellant_CM = self.propellant_I_23
+        from ..tools import (
+            parallel_axis_theorem_I11,
+            parallel_axis_theorem_I22,
+            parallel_axis_theorem_I33,
+            parallel_axis_theorem_I12,
+            parallel_axis_theorem_I13,
+            parallel_axis_theorem_I23,
+        )
+        from ..mathutils.vector_matrix import Vector
```

Comment:
please move these imports to the top of the file...
then you ran `make format` again